core/base/ripsPersistenceDiagram/RipsPersistenceDiagram.h

```diff
@@ -0,0 +1,57 @@
+/// \ingroup base
+/// \class ttk::RipsPersistenceDiagram
+/// \author Mattéo Clémot <matteo.clemot@univ-lyon1.fr>
+/// \date January 2024.
+///
+/// \brief TTK VTK-filter that computes the persistence diagram of a Rips
+/// complex.
+///
+/// \param Input Input table (vtkTable)
```

Comment:
I believe you can remove this line (the documentation of the `execute` function is doing the job and is more up to date).